torrt/rpc/deluge.py

```diff
@@ -92,7 +92,7 @@ def method_get_torrents(self, hashes=None):
         return result['torrents']
 
     def method_add_torrent(self, torrent, download_to=None):
-        torrent_dump = base64encode(torrent).decode('utf-8')
+        torrent_dump = base64encode(torrent)
```

Comment:
That'll break thins in py2.

Reply:
I ran add_torrent_from_url() with tox - and there were no compatibility issues found. Also `base64encode(torrent)`[ is referenced](https://github.com/idlesign/torrt/blob/b6d83e687056c62fe8dfea672efac98509397f5d/torrt/rpc/transmission.py#L91) in TransmissionRPC without `decode('utf-8')` . Can you cover it with test to point out the problem? Maybe it makes sense to move `decode('utf-8')` into `base64encode(torrent)`.